lib-ltc/include/ltcdisplayrgb.h

```diff
@@ -47,7 +47,7 @@ enum class LtcDisplayRgbColonBlinkMode {
 };
 
 enum class LtcDisplayRgbColourIndex {
-	DIGIT, COLON, MESSAGE, INFO, SOURCE, LAST
+	DIGIT, COLON, MESSAGE, INFO, SOURCE, RATE, LAST 
 };
```

Comment:
It is INFO -> ShowInfo(), SOURCE -> ShowSource(), hence RATE should be FPS -> ShowFPS()

Reply:
https://github.com/vanvught/rpidmx512/pull/159/commits/b0b599c5c9dcb500f569e622ef83472ec7191d6f